spec/spec_helper.rb

```diff
@@ -19,6 +19,11 @@
   add_filter "/spec/factories/"
   add_filter "/spec/rails_helper.rb"
   add_filter "/spec/spec_helper.rb"
+
+  # ignore files with fewer than 5 lines
+  add_filter do |source_file|
+    source_file.lines.count < 5
+  end
```

Comment:
@joshmcarthur Are you ok with this? It stops us failing coverage on the initial build because of the very short default Rails files e.g. 

![CleanShot 2022-01-27 at 11 15 04](https://user-images.githubusercontent.com/599867/151257233-7c31491f-5993-44cb-8d81-d734b169b527.png)